SA3/qa_web/templates/qa_web/base.html

```diff
@@ -66,6 +66,9 @@
 
     <!-- Latest compiled and minified Bootstrap JavaScript -->
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+    <!-- Markdown-it Js library -->
+    <script src="{% static 'js/markdown-it.min.js' %}"></script>
```

Comment:
If this is only necessary for the answerspage, shouldn't it be found inside that template instead of base?

Reply:
Actually QuestionIndex might need this to display question titles with markdown later on so I will leave it there for now.

Reply:
Good point